addons/hr_expense/static/src/views/kanban.xml

```diff
@@ -15,20 +15,27 @@
     </t>
 
     <t t-name="hr_expense.KanbanButtons" t-inherit="web.KanbanView.Buttons" t-inherit-mode="primary" owl="1">
-        <xpath expr="//t[@t-if='canCreate']" position="after">
-            <button type="button" class="d-none d-md-inline o_button_upload_expense btn btn-primary mx-1" t-on-click.prevent="uploadDocument">
-                Upload
-            </button>
-        </xpath>
-
-        <xpath expr="//t[@t-if='canCreate']" position="before">
-            <button type="button" class="d-inline d-md-none o_button_upload_expense btn btn-primary mx-1" t-on-click.prevent="uploadDocument">
-                Scan
-            </button>
-        </xpath>
-
-        <xpath expr="//div" position="inside">
-            <input type="file" name="ufile" class="d-none" t-ref="fileInput" multiple="1" accept="*" t-on-change="onChangeFileInput" />
+        <!-- Replace div to nest Main Actions toolbar inside Control Panel buttons and ensure consistency with list
+        buttons when adding a third button (Create Report) on mobile -->
```

Comment:
Not sure to get that :thinking: To me, with the old way of inheriting the template, we would still have put the buttons within the div control panel div https://github.com/odoo/odoo/blob/16.0/addons/web/static/src/views/kanban/kanban_controller.xml#L25

And with that, we could avoid duplicating the declaration of the create button that you have to do because of the "replace".

Why doesn't it work ? :thinking: